extide/gradle/src/org/netbeans/modules/gradle/api/execute/GradleCommandLine.java

```diff
@@ -82,42 +83,55 @@
      */
     public static final String CHECK_TASK = "check"; //NOI18N
 
+    public interface GradleOptionItem {
```

Comment:
Add `@since` + APIchange.

Reply:
Thanks!

Reply:
Apologies :( Also please bump spec version of the module. Not only GradleOptionItem, but more importantly the Flag and Parameter changed, so a module that uses the new symbol need to depend on new spec version to avoid potential issues if distributed through Plugin Portal.

Reply:
Well, the spec version increment has happened when Neil made the master ready for NB14 development. So 2.23 is pretty new, unreleased.